core/extraction/towns_fund_round_two.py

```diff
@@ -1370,7 +1371,7 @@ def convert_date(date_str: str) -> tuple[datetime, datetime]:
     return start_date, end_date
 
 
-def get_actual_forecast(row: pd.Series) -> str:
+def get_actual_forecast(row: pd.Series) -> str | float:
```

Comment:
is it possible to be a bit more explicit here and make that return type `str | np.nan`? I think int might be abit misleading

Reply:
I'm getting a type error when I change it to np.nan explicitly, as one is a type and the other isn't. I can substitute type(np.nan) instead, though not sure if that's standard practice

Reply:
ok my bad, I looked this up and float is correct for np.nan (albeit a little misleading and vague)